src/App/Models/Api/Request/TokenRequest.cs

```diff
@@ -24,7 +24,7 @@ public class TokenRequest
                 { "client_id", "mobile" }
             };
 
-            if(Device != null)
+            if(false && Device != null)
```

Comment:
Undo?

Reply:
Yes, I was testing the connection and I forgot it
I will correct it